_delphi_utils_python/setup.py

```diff
@@ -27,7 +27,7 @@
 
 setup(
     name="delphi_utils",
-    version="0.3.23",
+    version="0.3.24",
```

Comment:
you dont need to do this manually, the release process takes care of it for you!

https://github.com/cmu-delphi/covidcast-indicators/blob/0c65bb468d9aa3a6c2f37d6f28a5490029b32354/.github/workflows/create-release.yml#L41-L48